### PR TITLE
Creating Logger BasicConfig after importing the framework

### PR DESCRIPTION
**Pull request by mvsusp**
## Description of changes:

- Creating Logger BasicConfig after importing the framework

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker_containers/_modules.py [hunk 1]
```diff
@@ -24,7 +24,9 @@
 import boto3
 import six
 from six.moves.urllib.parse import urlparse
+import yaml
```
Comment (jesterhazy) [inline src/sagemaker_containers/_modules.py]: we do json for other config data. why introduce yaml?

Comment (jesterhazy) [reply src/sagemaker_containers/_modules.py]: oh i see, we are using it for formatting output. I think we should just stick with native dict repr.

Comment (mvsusp) [reply src/sagemaker_containers/_modules.py]: Agreed, removed PYAML.

File: src/sagemaker_containers/beta/framework/__init__.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 from sagemaker_containers import _errors as errors
 from sagemaker_containers import _env as env
 from sagemaker_containers import _functions as functions
+from sagemaker_containers import _logging as logging
 from sagemaker_containers import _mapping as mapping
 from sagemaker_containers import _modules as modules
```
Comment (jesterhazy) [inline src/sagemaker_containers/beta/framework/__init__.py]: change is out of scope for this PR, but its weird that we renamed all these modules _ and then alias them back to the old public name.

**Pull request merged**